Description: restored support for older automake versions by making 
  the inclusion of AM_PROG_AR in configure.ac conditional.
Applied-Upstream: commit:https://github.com/tschoonj/xraylib/commit/4a0f1d0f67f169c91d9c4b3561f4e44bff206373
Author: Tom Schoonjans <Tom.Schoonjans@me.com>

--- a/configure.ac
+++ b/configure.ac
@@ -37,7 +37,7 @@
 
 
 
-AM_PROG_AR
+m4_ifdef([AM_PROG_AR],[AM_PROG_AR])
 #at least version 2.0 of libtool is required for creating the fortran bindings
 LT_PREREQ([2.0.0])
 #dlopen is necessary for IDL bindings
